brainlit/algorithms/connect_fragments/dynamic_programming_viterbi.py

```diff
@@ -0,0 +1,420 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Feb 10 21:47:47 2021
+
+@author: frede
+"""
+
+import numpy as np
+from itertools import combinations
+from graspy.match import GraphMatch as GMP
```

Comment:
Is this import necessary? If so please add to requirements since it's not included currently e.g. see your docs [here](https://deploy-preview-223--brainlit.netlify.app/notebooks/algorithms/adaptive_thresh_tutorial.html)

Reply:
These aren't used, I can remove them. I'll also remove some of the other extraneous imports that weren't used.